funnel/templates/js/json_form.js.jinja2

```diff
@@ -0,0 +1,50 @@
+{% macro json_form_template() %}
+  {% raw %}
+    <script type="text/x-template" id="form-template">
+      <div>
+        <p v-if="title">{{ title }}</p>
+        <div v-for="field in jsonschema.fields">
+          <div class="mui-form__fields listwidget" :id="`field-${field.name}`">
+            <div class="mui-form__controls">
+
+              <div class="mui-select" v-if="field.type == 'string' && 'enum' in field && field.enum.length > 4">
+                <select :id="`${field.name}`">
+                  <option v-for="option in field.enum" :value="option">
+                    {{ option }}
+                  </option>
+                </select>
+                <label :for="`${field.name}`">{{ field.title }}</label>
+                <p class="mui-form__helptext" v-if="'description' in field">{{ field.description }}</p>
+              </div>
+
+              <div class="mui-textfield" v-if="field.type == 'string' && 'enum' in field">
+                <label :for="`${field.name}`" class="mui-form__label">{{ field.title }}</label>
+                <ul :id="`${field.name}`">
+                  <li v-for="option in field.enum">
+                    <input :id="`${option}`" :name="`${field.name}`" type="radio" :value="option">
+                    <label :for="`${option}`">{{ option }}</label>
+                  </li>
+                </ul>
+              </div>
+
+              <div class="mui-textfield mui-textfield--float-label" v-else-if="field.type == 'string'">
+                <input :id="`${field.name}`" type="text" :name="`${field.name}`" value="" />
```

Comment:
Don't use `field.name` for `id` as that is UGC — dangerous to insert in a page as it could override any other id. Use it only for `name`.

Reply:
`value` should use existing Rsvp form value if present.